PROCESS.md

```diff
@@ -0,0 +1,49 @@
+# Process
+
+## Preface
+
+This document is set out to describe the processes and guidelines for cloud based resource management related to CNCF credits and the CNCF credits program.
+
+A great deal is not automated or not yet possible to automate.
+
+repos:
+
+- [cncf/credits](https://github.com/cncf/credits) :: submissions, tickets and Terraform
+- [cncf-infra/aws-infra](https://github.com/cncf-infra/aws-infra) :: Terraform for AWS
+
+## Management
+
+Outline of management processes across cloud providers.
+
+### AWS
+
+Several CNCF projects consume AWS resources for several purposes, such as: testing, CI, hosting and more.
+
+#### Account creation
+
+New accounts are to be created in a structured form under org units for organization.
+
+```
+CNCF root account (org)
+├── [PROJECT] (org unit)
+│   ├── [PROJECT] (account)
+│   │   ├── [user email address] (iam user)
+│   │   ├── ...
+...
+```
+
+The provisioning of org units and accounts is performed manually, due to accounts requiring a level of interaction and the deletion of accounts with Terraform not being available (revisit this).
```

Comment:
Can you elaborate on this? Is there a gap in Terraform's ability to create `org unit` and `account` objects that prevents this part being done automatically?

Reply:
I believe there is a gap in the management of AWS accounts with deletion through Terraform, but it could change in the future and would be easily importable.

So for now, it would be create an account in the UI.
I haven't revisited this topic for a few months but don't mind checking it out again.